gno.land/pkg/sdk/vm/convert.go

```diff
@@ -153,6 +140,34 @@ func convertArgToGno(arg string, argT gno.Type) (tv gno.TypedValue) {
 			}
 			tv.SetUint64(u64)
 			return
+		case gno.Float32Type:
+			assertCharNotPlus(arg[0])
+			dec, _, err := apd.NewFromString(arg)
+			if err != nil {
+				panic(fmt.Sprintf("error parsing float32 %q: %v", arg, err))
+			}
+
+			f32, err := strconv.ParseFloat(dec.String(), 32)
+			if err != nil {
+				panic(fmt.Sprintf("error value exceeds float32 precision %q: %v", arg, err))
+			}
+
```

Comment:
```suggestion
```

how about getting rid this and other `newline`(s). Just to match the existing formatting style?

Reply:
Separating conditional statements with empty lines can help to make it more readable; most other cases don't have multiple conditionals and have less of a need for empty lines. I think adding empty lines should generally be encouraged.